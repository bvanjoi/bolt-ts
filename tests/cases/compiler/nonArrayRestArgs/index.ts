// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/nonArrayRestArgs.ts`, Apache-2.0 License

function foo(...rest: number) { // error
  //~^ ERROR: A rest parameter must be of an array type.
	var x: string = rest[0];
	return x;
}