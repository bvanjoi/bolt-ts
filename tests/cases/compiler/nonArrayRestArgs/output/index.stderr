  × A rest parameter must be of an array type.
   ╭─[../../tests/cases/compiler/nonArrayRestArgs/index.ts:3:14]
 3 │ ...rest: number
   · ───────────────
   ╰────
